### PR TITLE
feat(Site): return just non installable_apps for install app button, make migrate after install app

### PR DESCRIPTION
**Pull request by hamzaabadah**

### CODE REVIEW

File: bench_manager/bench_manager/doctype/site/site.py [hunk 1]
```diff
@@ -207,8 +208,9 @@ def get_installable_apps(doctype, docname):
 	with open(app_list_file, "r") as f:
 		apps = f.read().split("\n")
```
Comment (alaa-alsalehi) [inline bench_manager/bench_manager/doctype/site/site.py:209]: This function is really destruction where we using this?

File: bench_manager/bench_manager/doctype/site/site.py [hunk 2]
```diff
@@ -216,9 +216,10 @@ def get_installable_apps(doctype, docname):
 @frappe.whitelist()
 def get_removable_apps(doctype, docname):
 	verify_whitelisted_call()
-	removable_apps = frappe.get_doc(doctype, docname).app_list.split("\n")
-	removable_apps.remove("frappe")
-	return removable_apps
+	installed_apps = frappe.get_doc(doctype, docname).app_list.split("\n")
```
Comment (alaa-alsalehi) [inline bench_manager/bench_manager/doctype/site/site.py]: Where are we using this?

Comment (hamzaabadah) [reply bench_manager/bench_manager/doctype/site/site.py]: this return list for pop 
list contains apps installed on the site, and the user can uninstall one.

File: bench_manager/bench_manager/doctype/site/site.py (hunk 1 above)
Comment (hamzaabadah) [reply bench_manager/bench_manager/doctype/site/site.py:209]: this return list for pop button install the app
list contains apps not installed on the site, and the user can install one.

**Pull request merged**